content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+Setelah kamu menginisialisasi _control-plane_, kubelet akan berjalan normal.
+{{< /note >}}
+
+### Menginisialisasi simpul _control-plane_
+
+Simpul _control-plane_ adalah mesin dimana komponen-komponen _control plane_ berjalan, termasuk
+{{< glossary_tooltip term_id="etcd" >}} (basis data klaster) dan
+{{< glossary_tooltip text="peladen API" term_id="kube-apiserver" >}}
+(yang akan berkomunikasi dengan perangkat _command line_ {{< glossary_tooltip text="kubectl" term_id="kubectl" >}}.
+
+1. (Direkomendasikan) Jika kamu berencana untuk memperbarui klaster `kubeadm` _control-plane_ tunggal 
+menjadi ketersediaan tinggi kamu harus menentukan `--control-plane-endpoint` agar mengarah ke _endpoint_ yang digunakan bersama
+untuk semua simpul _control-plane_. _Endpoint_ tersebut dapat berupa nama DNS atau sebuah alamat IP dari _load-balancer_.
+1. Pilih _add-on_ jaringan Pod, dan pastikan apakah diperlukan argumen untuk
+diberikan pada `kubeadm init`. Tergantung
+penyedia pihak ketiga yang kamu pilih, kamu mungkin harus mengatur `--pod-network-cidr` dengan nilai
+yang spesifik pada penyedia tertentu. Lihat [Memasang _add-on_ jaringan Pod](#jaringan-pod).
+1. (Opsional) Sejak versi 1.14, `kubeadm` mencoba untuk mendeteksi _runtime_ kontainer pada Linux
+dengan menggunakan daftar _domain socket path_ yang umum diketahui. Untuk menggunakan _runtime_ kontainer yang berbeda atau
+jika ada lebih dari satu yang terpasang pada simpul yang digunakan, tentukan argumen `--cri-socket`
+pada `kubeadm init`. Lihat [Memasang _runtime_](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-runtime).
+1. (Opsional) Kecuali ditentukan sebelumnya, `kubeadm` akan menggunakan antarmuka jaringan yang diasosiasikan
+dengan _default gateway_ untuk mengatur alamat _advertise_ untuk peladen API pada simpul _control-plane_ ini.
+Untuk menggunakan antarmuka jaringan yang berbeda, tentukan argumen `--apiserver-advertise-address=<ip-address>` 
+pada `kubeadm init`. Untuk menyebarkan klaster Kubernetes IPv6 menggunakan pengalamatan IPv6, kamu
+harus menentukan alamat IPv6, sebagai contoh `--apiserver-advertise-address=fd00::101`
+1. (Opsional) Jalankan `kubeadm config images pull` sebelum `kubeadm init` untuk memastikan
+konektivitas ke registri _image_ kontainer gcr.io.
+
+Untuk menginisialisasi simpul _control-plane_ jalankan:
+
+```bash
+kubeadm init <args>
+```
+
+### Pertimbangan mengenai apiserver-advertise-address dan ControlPlaneEndpoint
+
+Meski `--apiserver-advertise-address` dapat digunakan untuk mengatur alamat _advertise_ untuk peladen
+API pada simpul _control-plane_ ini, `--control-plane-endpoint` dapat digunakan untuk mengatur _endpoint_ yang digunakan bersama
+untuk seluruh simpul _control-plane_.
+
+`--control-plane-endpoint` tidak hanya mengizinkan alamat IP tetapi juga nama DNS yang dapat dipetakan pada alamat IP.
+Silakan hubungi _network administrator_ untuk mengevaluasi solusi-solusi yang mempertimbangkan pemetaan tersebut.
```

Comment:
```suggestion
Silakan hubungi administrator jaringan kamu untuk mengevaluasi solusi-solusi yang mempertimbangkan pemetaan tersebut.
```

https://kbbi.kemdikbud.go.id/entri/administrator

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc